.github/workflows/terraform-apply.yml

```
@@ -0,0 +1,173 @@
+name: Terraform Apply
+
+on:
+  workflow_call:
+    
+    inputs:
+      environment:
+        required: true
+        type: string
+      apply_args:
+        required: false
+        type: string
+        description: Arguments provided to the apply command, e.g. terraform apply -var-file=dev.tfvars
+      terraform_dir:
+        required: false
+        type: string
+        default: terraform
+        description: Path to directory, where terraform is situated relative to the repo root. Should not have trailing slash.
+      plan_artifact:
+        required: false
+        type: boolean
+        default: true
+        description: Look for artifact with the name "<environment>_plan", which is used by the apply command to save time executing the flow.
+      assume_profile_arn:
+        required: false
+        type: string
+        description: If provided, this profile will be assumed before executing the terraform.
+      comment_to_pr:
+        required: false
+        type: boolean
+        default: true
+      
+    secrets:
+      AWS_ACCESS_KEY_ID:
+        required: true
+      AWS_SECRET_ACCESS_KEY:
+        required: true
+      TERRAFORM_CLOUD:
+        required: true
+      SSH_KEY:
+        required: false
+
+
+jobs:
+  terraform-apply:
+    runs-on: ubuntu-latest    
+    defaults:
+      run:
+        working-directory: ${{ inputs.terraform_dir }}
+    steps:
+    - uses: actions/checkout@v3
+    - uses: hashicorp/setup-terraform@v2
+
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v1
```

```suggestion
      uses: aws-actions/configure-aws-credentials@v1-node16
```
Use node16 version to avoid warnings